Week3/moreJs-6.js

```diff
@@ -0,0 +1,19 @@
+/*
+Create a function called vehicle, like before,
+ but takes another parameter called age, 
+ so that vehicle("blue", 1, 5) 
+ prints 'a blue used car'
+*/
+
+function vehicle (color, type, age){
+    let types = [' car' , ' motorbike'];
+    console.log('a ' + color +' ' + condition(age) + types[(type-1)]);
+}
+function condition(years){
```

Comment:
Maybe the name `age` here is more consistent, because you use it aswell above

Reply:
yes. you re right. 
I was trying to find some other parameters name. I thought it might confuse ('age' in 2 places).
but I forgot about scopes, and parameters I think.
I changed it to age. its better